ott/tools/gaussian_mixture/scale_tril.py

```diff
@@ -157,24 +157,40 @@ def _flatten_cov(cov: jnp.ndarray) -> jnp.ndarray:
     return (cost_fn.norm(x0) + cost_fn.norm(x1) +
             cost_fn.pairwise(x0, x1))[...,]
 
-  def transport(
-      self, dest_scale: 'ScaleTriL', points: jnp.ndarray
-  ) -> jnp.ndarray:
-    """Transport between 0-mean normal w/ current scale to one w/ dest_scale.
+  def transport_scale_matrix(self, dest_scale: 'ScaleTriL') -> jnp.ndarray:
```

Comment:
maybe transport / matrix are a bit self-evident for this naming. monge_map ? (ref can be given for instance to https://arxiv.org/pdf/1712.01504.pdf p.9)